hibernate-core/src/test/java/org/hibernate/jpa/test/lock/QueryLockingTest.java

```diff
@@ -170,6 +171,7 @@ public void testPessimisticForcedIncrementOverall() {
 	}
 
 	@Test
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with `@SkipForDialect( value = CockroachDB192Dialect.class`

Reply:
done